src/ripple/shamap/impl/SHAMapTreeNode.cpp

```diff
@@ -300,15 +311,15 @@ bool SHAMapTreeNode::updateHash ()
         s.add32 (HashPrefix::leafNode);
         s.addRaw (mItem->peekData ());
         s.add256 (mItem->getTag ());
-        nh = s.getSHA512Half ();
+        nh = sha512Half(HashPrefix::leafNode,
+            make_Slice(mItem->peekData()),
+                mItem->getTag());
```

Comment:
You can remove the `Serializer s` here, because it's block-scoped, and no longer used.